neo4j-browser-5.26.yaml

```diff
@@ -0,0 +1,54 @@
+package:
+  name: neo4j-browser-5.26
+  version: 5.26.5
+  epoch: 0
+  description: Neo4j Browser is the general purpose user interface for working with Neo4j.
+  copyright:
+    - license: GPL-3.0-or-later
+
+environment:
+  environment:
+    NODE_OPTIONS: "--openssl-legacy-provider"
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - maven
+      - nodejs-16
+      - openjdk-17-default-jdk
+      - openssl-provider-legacy
+      - posix-libc-utils # getconf
+      - yarn
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/neo4j/neo4j-browser
+      tag: neo4j-${{package.version}}
+      expected-commit: 6011f52e65c77ab775a8f6e7ceac8d4e6d5a5825
+
+  - runs: |
+      export NODE_OPTIONS=--openssl-legacy-provider
+      yarn install --frozen-lockfile
+      yarn build
+      node ./scripts/prepare-mvn-package.js
+      mvn package -DskipTests=true
+      mkdir -p ${{targets.contextdir}}/var/lib/neo4j/lib/
+      find target/ -maxdepth 1 -type f -name 'neo4j-browser-*.jar' ! -name '*-tests.jar' -exec mv {} ${{targets.contextdir}}/var/lib/neo4j/lib/ \;
+
+  - uses: strip
+
+# this is a dependency to main neo4j package so please update it first to unblock neo4j main package.
+update:
+  enabled: true
+  github:
+    identifier: neo4j/neo4j-browser
+    use-tag: true
+    strip-prefix: neo4j-
+    tag-filter: 5.26.
```

Comment:
Nobody's going to know to bump this. Could we have neo4j depend on neo4j-browser~${{package.version}}? Or even the major/minor version?

Reply:
RJ, looks like this is not supported. bot should be good to handle the updates here looking at other examples like cert-manager. 
```yaml
tag-filter: ${{vars.major-minor-version}}.
```

Reply:
Let's remove the tag filter here, and pin at runtime in neo4j after this has built